maps-compose/src/main/java/com/google/maps/android/compose/GoogleMapComposable.kt

```diff
@@ -0,0 +1,18 @@
+package com.google.maps.android.compose
+
+import androidx.compose.runtime.ComposableTargetMarker
+
+/**
+ * An annotation that can be used to mark a composable function as being expected to be use in a
+ * composable function that is also marked or inferred to be marked as a [GoogleMapComposable].
```

Comment:
I'd suggest adding in the Javadoc the result of using the annotation, which you have in the PR comment. Something like:

>This will produce build warnings when Maps Compose composable functions are used outside of the GoogleMap content lambda. Similarly, a warning is shown when non-Maps Compose composables are used within the GoogleMap content lambda.

Reply:
Done in 2f27a84